v2/auth/cachedauth/sign_in.go

```diff
@@ -11,9 +11,11 @@ import (
 )
 
 var lock sync.RWMutex
-var tokens auth.Tokens
 
 var config *Config
+var tokens map[string]auth.Tokens
+
+const latest = "latest"
```

Comment:
There will be no collisions with valid usernames since all usernames are emails?

Reply:
exactly